Release 3.5.0 (#1295)
uccser/cs-field-guide: docs/source/changelog.rst

We have listed major changes for each release below.

3.5.0
==============================================================================

- **Release date:** 7th July 2020
- **Downloads:** `Source downloads are available on GitHub <https://github.com/uccser/cs-field-guide/releases/tag/3.5.0>`__

**Changelog:**

> - Add ability to show colour codes in Hexadecimal on the Pixel Viewer interactive. <https://github.com/uccser/cs-field-guide/issues/1277>`__

Reviewer: ```suggestion
- Add hexadecimal options for the Pixel Viewer interactive. <https://github.com/uccser/cs-field-guide/issues/1277>`__
```

> - Add Hexadecimal version of colour mixer interactives. <https://github.com/uccser/cs-field-guide/issues/1290>`__

Reviewer: ```suggestion
- Add hexadecimal option for the colour mixer interactives. <https://github.com/uccser/cs-field-guide/issues/1290>`__
```
